generators/app/templates/src/StarterKit/Shared/Swagger/LowerCaseQueryAndBodyParameterFilter.cs

```diff
@@ -1,19 +1,18 @@
-using Swashbuckle.AspNetCore.Swagger;
-using Swashbuckle.AspNetCore.SwaggerGen;
-using System;
 using System.Linq;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
 
 namespace StarterKit.Shared.Swagger
 {
     public class LowerCaseQueryAndBodyParameterFilter : IOperationFilter
```

Comment:
Indien ik mij niet vergis worden nu enkel de query parameters naar lowerCase gezet. Het lijkt mij verstandig om de body ook op te nemen of op zijn minst de klasse naam aan te passen om dit te reflecteren.